### PR TITLE
Improved description of fingerprint handling in command line wiki

### PR DESCRIPTION
**Pull request by albertony**

### CODE REVIEW

File: Command-Line.md [hunk 1]
```diff
@@ -583,35 +583,36 @@ Clients must contain the following file:
...
+change the paths in the example accordingly.
+
+```
+MKDIR "%LocalAppData%\Barrier\SSL\Fingerprints" >NUL 2>&1
+SET OPENSSL_CONF=C:\Program Files\Barrier\barrier.conf
 SET RANDFILE=%LocalAppData%\Barrier\SSL\.rnd
 "C:\Program Files\Barrier\openssl.exe" req -x509 -nodes -days 365 -subj /CN=Barrier -newkey rsa:2048 -keyout "%LocalAppData%\Barrier\SSL\Barrier.pem" -out "%LocalAppData%\Barrier\SSL\Barrier.pem"
 IF EXIST "%RANDFILE%" DEL "%RANDFILE%"
-"C:\Program Files\Barrier\openssl.exe" x509 -fingerprint -sha1 -noout -in "%LocalAppData%\Barrier\SSL\Barrier.pem" > "%LocalAppData%\Barrier\SSL\Fingerprints\Local.txt"
+FOR /F "tokens=2 delims=^=" %a in ('""C:\Program Files\Barrier\openssl.exe" x509 -fingerprint -sha1 -noout -in "%LocalAppData%\Barrier\SSL\Barrier.pem""') DO ECHO %a > "%LocalAppData%\Barrier\SSL\Fingerprints\Local.txt"
```
Comment (shymega) [inline Command-Line.md:607]: Why do we need this...? What does it add to the previous description?

**Comment by shymega** [comment]
Reviewing, and asking @p12tic to review too. Other than that, looks OK...

**Comment by shymega** [comment]
Apart from my one comment questioning, it looks great.

File: Command-Line.md (hunk 1 above)
Comment (albertony) [reply Command-Line.md:607]: This is to strip off the `SHA1 Fingerprint=`, to make the Local.txt content identical to what barrier produces, and then I did not have to specify in the description to only copy the hash and not this prefix when adding to TrustedServers.txt. I know, it looks a bit "verbose"..

I'm considering rewriting this section, even more. Perhaps with PowerShell statements, which is probably a bit easier example to read, or maybe instead go back and just mention the plain openssl commands without the other stuff. I haven't decided yet. I have been experimenting with different solutions (thats why the late response). For instance on how I can use `openssl s_client` on the client side to grab the fingerprint from remote server. Will probably take some time (at least days, perhaps some weeks) for me to finish and conclude, so its up to you if you want to merge this one, else I can make it a draft/work-in-progress kind of PR, or even delete it, until next step is ready.

**Comment by p12tic** [comment]
Looks great, thank you!

**Comment by shymega** [comment]
@p12tic FYI, you didn't actually merge this into the Barrier repo's wiki. I have merged it now.

**Pull request merged**